### PR TITLE
Misc documentation updates

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind documentation
#### What this PR does / why we need it:
more details in each commit

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

**Comment by mrunalp** [comment]
Looks great! Will let others review as well before we tag it.

**Comment by haircommander** [comment]
In accordance with the policy as defined, this PR has a week voting period where folks can +1/-1 it before we merge

@nalind @giuseppe @umohnani8 @saschagrunert @fidencio @kolyshkin PTAL

File: GOVERNANCE.md
Comment (lenicatko) [inline GOVERNANCE.md:37]: This might be a good place to add also things that you don't mention in reviewer/maintainer sections and encourage users to participate more in the community, and become contributor, like helping with improving documentation, or improving website, evangelize the project by writing blog, etc. CNCF has good examples in [their template under community participant](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md#community-participant)

File: GOVERNANCE.md
Comment (jrvaldes) [inline GOVERNANCE.md:5]: This section is no longer present in the proposed structure of the document. I suggest removing it or explicitly adding the subsections of `Contribution Tiers`.

**Review by jrvaldes — COMMENTED** [review]
@haircommander Thanks for working on this.  I added a suggestion, PTAL.

File: GOVERNANCE.md
Comment (jrvaldes) [inline GOVERNANCE.md:45]: Looks like the `membership.yml` template is no longer there, use the following URL to leverage the current template.

```
https://github.com/cri-o/cri-o/issues/new?template=reviewer.yml&title=REQUEST%3A+New+reviewer+status+for+%3Cyour-GH-handle%3E
```

**Comment by haircommander** [comment]
great catches @jrvaldes @lenicatko, thanks! I have adopted your feedback and squashed the changes

**Comment by jrvaldes** [comment]
LTGM

**Comment by haircommander** [comment]
@nalind @giuseppe @umohnani8 @saschagrunert @fidencio @kolyshkin PTAL

File: GOVERNANCE.md
Comment (TomSweeneyRedHat) [inline GOVERNANCE.md:109]:
```suggestion
Through the mechanism of the OWNERS file, it’s possible to define sub-tree approvers for areas of specialty. One way this happens is if a section of code that a sub-tree approver is responsible for has a security vulnerability, they will be included in the discussions. However, they need not necessarily be included if there is no such vulnerability. For the purposes of this document, when “approvers” is mentioned, sub-tree approvers are not automatically included, though they could be.
```

Comment (TomSweeneyRedHat) [reply GOVERNANCE.md:109]: and maybe: "when "approvers" is" to "when the word "approvers" is"

File: GOVERNANCE.md
Comment (TomSweeneyRedHat) [inline GOVERNANCE.md:121]: two years seems too long.  I'd go a year, maybe 6 months.  YMMV

File: GOVERNANCE.md
Comment (TomSweeneyRedHat) [inline GOVERNANCE.md:169]:
```suggestion
There is an alias on GitHub that one can use to ping all of the CRI-O reviewers and approvers: "@cri-o/cri-o-maintainers".
```

File: GOVERNANCE.md
Comment (TomSweeneyRedHat) [inline GOVERNANCE.md:185]: maybe add links to each of these?

File: GOVERNANCE.md
Comment (haircommander) [reply GOVERNANCE.md:109]: I have dropped the security section as upon reading again I found it clunky

File: GOVERNANCE.md
Comment (haircommander) [reply GOVERNANCE.md:121]: I will drop to one

**Comment by haircommander** [comment]
In addition to addressing @TomSweeneyRedHat 's comments, I have also added a new commit adding a new tier of contributor: org member, which is a position that currently exists but was not documented, nor was the process of becoming one defined. This was done by inspiration from a [slack thread](https://kubernetes.slack.com/archives/CAZH62UR1/p1673696055595509), thanks Shunli (whose github handle I don't know yet)

File: roadmap.md
Comment (littlejawa) [inline roadmap.md:28]: There are multiple references of SIG-Node, but no link to it. Maybe adding a link here (or at the first occurrence of "SIG-Node") would be nice to help track those initiatives?

Comment (haircommander) [reply roadmap.md]: I added a couple of links embellishing references I made, PTAL @littlejawa :)

**Review by littlejawa — APPROVED** [review]

**Review by saschagrunert — APPROVED** [review]
LGTM

**Pull request merged**
